examples/logto/libraries/logto.ts

```diff
@@ -0,0 +1,10 @@
+import LogtoClient from '@logto/next'
+
+export const logtoClient = new LogtoClient({
+  appId: '<your-application-id>',
+  appSecret: '<your-app-secret-copied-from-console>',
+  endpoint: '<your-logto-endpoint>', // E.g. http://localhost:3001
+  baseUrl: '<your-nextjs-app-base-url>', // E.g. http://localhost:3000
+  cookieSecret: 'complex_password_at_least_32_characters_long',
```

Comment:
I think it would make sense if these were read from a `.env.local.example` file. Could you add one?

Reply:
updated